state-indexer/src/configs.rs

```diff
@@ -386,6 +395,7 @@ impl ScyllaStorageManager for ScyllaDBManager {
                 "INSERT INTO state_indexer.account_access_keys
                     (account_id, block_height, active_access_keys)
                     VALUES(?, ?, ?)",
+                None,
```

Comment:
Yeah, we want. This is a leftover since I was focused on `tx-indexer` and decided to use the default value for others. Thank you for bringing this up. I'll adjust.